docs/reference-architectures/power-platform/eventually-consistent-instances.md

```diff
@@ -0,0 +1,44 @@
+
+# Eventual Consistency in Power Platform
+
+Strategically create data using an upsert to allow systems to synchronise data. Guarantee that the consuming application will be able to accept requests when the data is awaiting synchronisation or missing.
+
+This pattern can be useful in situations when:
+- The system that sends reference data is down.
+- The synchronisation of data takes a long time or the process is delayed.
+- Consuming systems have no logic on the creation of the entity being created. 
+
+## Context and problem
+
+In many modern applications it is vital to consider and implement fault tolerance. In certain situations you may synchronise accounts and contacts for example from one instance of power platform to another. Suppose you have two instances of Dynamics 365 and "Instance A" synchronises data to "Instance B", and another system which reads data from "Instance A" and sends a payload with unique identifiers or alternate keys to "Instance B". When "Instance B" does not have the data, the user will receive a bad request as the entity with that record does not exist.
+
+The following examples show the potential journies for a record submission. 
+
+**Example 1** - Happy path and all working as expected.
+![Example of multiple system synchronisation](./_images/data-dependent-example.png)
```

Comment:
could we please improve the accessibility of this diagram? i.e. you could use white background so they are visible when using dark mode.